tsconfig.esm.json

```diff
@@ -1,8 +1,9 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
-    "module": "esnext",
+    "module": "commonjs",
```

Comment:
Change it to `esnext`. You can `import` `@coolio/http` from `dist/cjs` if esnext doesn't work correctly. In the future I'll try to export these in more user-friendly manner.